File: Data Structures & Algorithms/duplicate-integer/submission-3.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        int a=0;
        for(int i =0; i<nums.length-1;  i++){
            for(int j=i+1; j<nums.length;j++){
                if(nums[i]==nums[j]) a++;
            }
        }
        return a>0;
        
    
}

}

File: Data Structures & Algorithms/search-insert-position/submission-3.py
class Solution:
    def searchInsert(self, nums: List[int], target: int) -> int:
        l = 0
        r = len(nums)-1
        m = 0
        while l<=r:
            m = l + (r-l)//2
            if nums[m] == target:
                return m
            elif nums[m]< target:
                l = m+1
            else:
                r = m-1
        return m+1 if nums[m] < target else m